tutorial_dataset_1.qmd

```diff
@@ -435,7 +435,7 @@ source("build.R")
 As a final step, build a report for the study
 
 ```{r, eval=FALSE}
-traits.build_database$build_info$version <- "4.0.0"  # a fix because the function was built around specific AusTraits versions
+traits.build_database$build_info$version <- "5.0.0"  # a fix because the function was built around specific AusTraits versions
```

Comment:
What does `traits.build_database$build_info$version` evaluate to before rewriting to "5.0.0"? Will this be fixed eventually?